### PR TITLE
Switch to literal string interpolation

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: stratis-storage/project#456

### CODE REVIEW

File: src/stratis_cli/_actions/_pool.py [hunk 1]
```diff
@@ -268,10 +267,11 @@ def init_cache(namespace):  # pylint: disable=too-many-locals
...
-                    "to pool %s but stratisd reports that it did not actually "
-                    "add some or all of the blockdevs requested; devices "
-                    "added: (%s), devices requested: (%s)"
+                    f"Expected to add the specified blockdevs as cache to pool "
+                    f"{namespace.pool_name} but stratisd reports that it did not "
+                    f"actually add some or all of the blockdevs requested; "
+                    f"devices added: ({', '.join(devnodes_added)}), "
+                    f"devices requested: ({', '.join(blockdevs)})"
                 )
                 % (namespace.pool_name, ", ".join(devnodes_added), ", ".join(blockdevs))
```
Comment (bmr-cymru) [inline src/stratis_cli/_actions/_pool.py:276]: Missing deletion of the format line?

Comment (bgurney-rh) [reply src/stratis_cli/_actions/_pool.py]: Yes; sorry.  I thought I deleted that, but there must have been two before.

**Comment by bgurney-rh** [comment]
Somewhere along the way, the `make coverage` test failed; I need to fix that at some point.

**Comment by bgurney-rh** [comment]
That was a lucky guess.  Something in that last commit, with a small amount of changes...
```
 src/stratis_cli/_actions/_data.py             | 2 +-
 src/stratis_cli/_actions/_stratis.py          | 2 +-
 src/stratis_cli/_actions/_stratisd_version.py | 4 ++--
 src/stratis_cli/_actions/_utils.py            | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)
```
...broke the coverage test.

**Pull request closed without merging**